client/components/site-selector/index.jsx

```diff
@@ -191,6 +193,15 @@ export class SiteSelector extends Component {
 	};
 
 	onSiteSelect = ( event, siteId ) => {
+		if ( siteId !== ALL_SITES ) {
+			const visibleSites = this.visibleSites.filter( ( ID ) => ID !== ALL_SITES );
+			this.props.recordTracksEvent( 'calypso_switch_site_click_item', {
+				position: visibleSites.indexOf( siteId ),
+				list_item_count: visibleSites.length,
+				is_searching: this.state.searchTerm.trim().length > 0,
```

Comment:
A user could search by multiple spaces. I suggest to remove the `trim()`.
```suggestion
				is_searching: this.state.searchTerm.length > 0,
```

Reply:
> A user could search by multiple spaces.

Who does that?!?!

Reply:
Thanks for accepting the change! 👌